extide/gradle/nbproject/org-netbeans-modules-gradle.sig

```diff
@@ -1,1074 +0,0 @@
-#Signature file v4.1
-#Version 2.0
-
```

Comment:
I don't think you want to completely remove the file. You may want to remove just the lines that are no longer present.